### PR TITLE
Enable bandwidth-manager by default for new deployments

### PR DESCRIPTION
**Pull request by qmonnet**
For Cilium v1.10, bandwidth manager is no longer considered as a beta feature: re-enable it by default for new deployments.

This PR reverts the commit that disabled the feature by default, but it does not update the documentation that still marks the bandwidth manager as _beta_.

Reverts part of https://github.com/cilium/cilium/pull/13515.
Also related: Follow-ups for EDT https://github.com/cilium/cilium/issues/13371.

### CODE REVIEW

**Comment by qmonnet** [comment]
test-me-please

**Review by christarazi — APPROVED** [review]

**Review by sayboras — APPROVED** [review]

**Review by kkourt — APPROVED** [review]

**Comment by qmonnet** [comment]
test-me-please

File: install/kubernetes/cilium/values.yaml [hunk 1]
```diff
@@ -173,7 +173,7 @@ azure:
   # userAssignedIdentityID: 00000000-0000-0000-0000-000000000000
 
 # TODO: Add documentation
-bandwidthManager: false
+bandwidthManager: true
```
Comment (christarazi) [inline install/kubernetes/cilium/values.yaml:178]: Would this be the time to add documentation?

**Review by christarazi — APPROVED** [review]

**Comment by christarazi** [comment]
test-me-please

File: install/kubernetes/cilium/values.yaml (hunk 1 above)
Comment (qmonnet) [reply install/kubernetes/cilium/values.yaml:178]: Why not, this commit is a revert but I can add a new one to comment the value. Thanks!

**Comment by qmonnet** [comment]
test-me-please

**Review by christarazi — APPROVED** [review]

**Pull request merged**